dialog-polyfill.js

```diff
@@ -723,20 +726,10 @@
         dialog.close();
       }
       dialogPolyfill.formSubmitter = null;
+
     }, true);
   }
 
-  dialogPolyfill['forceRegisterDialog'] = dialogPolyfill.forceRegisterDialog;
```

Comment:
FWIW this is for Closure Compiler's advanced compilation support, otherwise these methods get renamed. Having said that some naïve testing now seems to imply it hasn't worked properly for a while anyway.

I'm not against removing this but in case you were wondering why we had this code.